<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterpolateLabelsDialog</class>
 <widget class="QDialog" name="InterpolateLabelsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interpolate Labels - ITK-SNAP</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:16px;</string>
   </property>
   <property name="text">
    <string>Interpolation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>34</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:12px;
color: gray;</string>
   </property>
   <property name="text">
    <string>Use this tool to fill in sparsely drawn segmentations. For example, you can label a structure on every fifth slice and fill in the gaps using this tool. You can also create three-dimensional scaffolds and fill in the space in between.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>100</y>
     <width>314</width>
     <height>158</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QRadioButton" name="btnInterpolateAll">
      <property name="toolTip">
       <string>All labels in the segmentation will be interpolated at the same time</string>
      </property>
      <property name="text">
       <string>Interpolate all labels</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="btnInterpolateOne">
      <property name="toolTip">
       <string>A single label in the segmentation will be interpolated.</string>
      </property>
      <property name="text">
       <string>Interpolate a single label</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Label to interpolate:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="inLabelToInterpolate">
      <property name="toolTip">
       <string>Select the label which will be interpolated. Your segmentation should have two or more slices painted with that label.</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Paint with label:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="inActiveLabel">
      <property name="toolTip">
       <string>Select the label which will be used to fill in the interpolated voxels. Normally this is the same label as the &quot;label to interpolate&quot; but you have the option of using a different label.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Paint over label(s):</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="inDrawOverFilter">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select which label of set of labels will be painted over during interpolation. You can paint over all labels, clear voxels, or a specific label.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>600</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInterpolate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>600</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Interpolate</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="morphoptions">
    <widget class="QGroupBox" name="MorphOptions">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>145</height>
      </rect>
     </property>
     <property name="title">
      <string>Advanced Options:</string>
     </property>
     <widget class="QCheckBox" name="chkMorphologyUseDistance">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>199</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Use an alternative signed distance function-based method.  Slightly faster, but can fail for twisted shapes.</string>
      </property>
      <property name="text">
       <string>Use signed distance function</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkMorphologyInterpolateOneAxis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>202</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Interpolate slices along a single axis. By default, interpolation is applied along all three axes in the image.</string>
      </property>
      <property name="text">
       <string>Interpolate along a single axis</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>84</y>
        <width>291</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>80</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QComboBox" name="morphologyInterpolationAxis">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>251</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Axial</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sagittal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Coronal</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="chkMorphologyUseOptimalAlignment">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Use optimal slice alignment instead of the default heuristic.  This can take a long time.</string>
      </property>
      <property name="text">
       <string>Use optimal slice alignment</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="bwaoptions">
    <widget class="QGroupBox" name="BWAoptions">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Advanced Options:</string>
     </property>
     <widget class="QCheckBox" name="chkBWAInterpolateIntermediateOnly">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Interpolate intermediate slices only</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkBWAUseContourOnly">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Use binary weighted averaging only</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkSliceDirection">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Manually select interpolation axis</string>
      </property>
     </widget>
     <widget class="QComboBox" name="SliceDirectionAxis">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>251</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Axial</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sagittal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Coronal</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="chkBWAOverwriteSegmentation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Overwrite existing segmentation</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QRadioButton" name="btnBinaryWeightedAverage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Binary Weighted Averaging with Random Forest</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">InterpolationMethods</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="btnMorphological">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>189</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Morphological Interpolation</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">InterpolationMethods</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Interpolation Method:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnInterpolateAll</sender>
   <signal>toggled(bool)</signal>
   <receiver>inLabelToInterpolate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnInterpolateAll</sender>
   <signal>toggled(bool)</signal>
   <receiver>inActiveLabel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMorphologyInterpolateOneAxis</sender>
   <signal>toggled(bool)</signal>
   <receiver>morphologyInterpolationAxis</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSliceDirection</sender>
   <signal>toggled(bool)</signal>
   <receiver>SliceDirectionAxis</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>651</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>InterpolationMethods</sender>
   <signal>idToggled(int,bool)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="InterpolationMethods"/>
 </buttongroups>
</ui>
